### DISCUSSION TITLE
fallback image for $media('thumb',['class'=>'foo'])

### DISCUSSION THREAD

**Discussion by jayenne** [spatie/laravel-medialibrary#2508] [Q&A]
Hi, I see you have fallback urls and paths but can they be used when returning collections/convertions/responsive where no media exists? (either not uploaded or seeder/populated)
*I don't expect them to actually be responsive obvs

I have a problem where my app fails if no avatar exists on `$data->user->getFirstMedia('avatar')->img('responsive')->attributes(['class'=>'w-8 h-8 rouded-full'])` and would like it to fail gracefully using the fallback image instead of breaking.

or Is there a better eay to acheieve this already?

**Comment by lloricode** [comment]
+1
